internal/config/upsun-cli.yaml

```
@@ -0,0 +1,78 @@
+# Upsun CLI configuration
+#
+# Based on the legacy CLI config in https://github.com/platformsh/legacy-cli/blob/main/config.yaml
+#
+# See ../internal/config/schema.go for the required keys
+wrapper:
+  homebrew_tap: platformsh/tap/upsun-cli
+  github_repo: platformsh/cli
+
+application:
+  name: "Upsun CLI"
+  slug: "upsun"
+  executable: "upsun"
+  env_prefix: "UPSUN_CLI_"
+  user_config_dir: ".upsun-cli"
+
+  installer_url: "https://cli.upsun.com/installer"
+  manifest_url: "https://cli.upsun.com/manifest.json"
+
+  wrapped_disabled_commands:
```

```suggestion
  disabled_commands:
```

we probably don't want these whether wrapped or not?